core/sail/lmdb/src/main/java/org/eclipse/rdf4j/sail/lmdb/ValueStore.java

```diff
@@ -74,19 +74,19 @@
 	 * Constants *
 	 *-----------*/
 
-	private static final byte ID_KEY = 0x0;
+	private static final byte URI_VALUE = 0x0; // 00
```

Comment:
Maybe use binary literals and make it proper bit fields? 
```java
private static final byte LITERAL_VALUE = 0b00 ;
private static final byte BNODE_VALUE   = 0b01 ;
private static final byte NAMESPACE_VALUE = 0b011 ;
private static final byte ID_KEY = 0b0100 ;
private static final byte HASH_KEY   =0b0101 ;
private static final byte HASHID_KEY=0b0110 ;
```
I find this easier to when looking at bit patterns hiding in a larger primitive object.

Reply:
We can change this. I just find it a bit harder to determine the order of binary encoded numbers.